tests/data/test_history.py

```diff
@@ -657,6 +669,12 @@ def test_jsondatahandler_ohlcv_load(testdatadir, caplog):
     df = dh.ohlcv_load('XRP/ETH', '5m')
     assert len(df) == 711
 
+    df_mark = dh.ohlcv_load('XRP/USDT', '1h', candle_type="mark")
+    assert len(df_mark) == 99
```

Comment:
I'm kind of confused why `len(df_mark)` is `99`, I feel like it should be 100, because there's 100 array items in `XRP_USDT-1h-mark.json`, but when I test it, it tells me that `len(df_mark)` is equal to `99`

Reply:
last candle is assumed incomplete and dropped.
I'm pretty well aware that this dropping happens twice (once when downloading, once when loading) .... but it's so far not been worth fixing, and i wouldn't try a fix for this as part of feat/short (if at all, in develop - but it's not a big deal i think).